cvat-ui/src/utils/mousetrap-react.tsx

```diff
@@ -15,6 +15,7 @@ export interface KeyMapItem {
     scope: ShortcutScope;
     nonActive?: boolean;
     applicable?: string[];
+    weight?: number;
```

Comment:
You've added `weight` but it equals to 10 for all added shortcuts, so, what is was made to?

Reply:
If shortcut doesnt have weight it is considered as 0. So this group of shortcuts is rendered in the bottom of standard workspace controls category, thats why it was added(by default  they were on top of it before `cancel` for example). 

The fact that they have same weight doesnt really matter because it doesnt affect ordering inside of this group. They still will be at the bottom of the category. On the other side it may be a bit confusing so Ive changed it to different values.